File: help/eng_gb/exercicios_da_ficha_6/Ficha06_002/Ficha06_002.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha06_002 {

    //Main Function
    static void Main() {
        Console.Write("Height : " );
        int height = int.Parse(Console.ReadLine());
        for( int i = 0; i < height; i = i + 1 ) {
            for( int j = 0; j < height; j = j + 1 ) {
                Console.Write("*" );
            }
            Console.Write("\n" );
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 21:55

File: help/eng_gb/exercicios_da_ficha_6/Ficha06_002/Ficha06_002.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Height : " );// print text
    height = parseInt(prompt("Height : " ,"0") );
    
    document.write(height + "\n");// print input 
    for( var i = 0; i < height; i = i + 1 ) {
        for( var j = 0; j < height; j = j + 1 ) {
            document.write("*" );
        }
        document.write("\n" );
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 21:55
